### PR TITLE
emboss: fix perl usage and add requirement

### PR DESCRIPTION
**Pull request by bernt-matthias**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/emboss_5/emboss_cpgplot_wrapper.pl [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#! /usr/bin/perl -w
+#!/usr/local/bin/env perl
```
Comment (nsoranzo) [inline tools/emboss_5/emboss_cpgplot_wrapper.pl]:
```suggestion
#!/usr/bin/env perl
```

File: tools/emboss_5/emboss_infoseq_wrapper.pl [hunk 2]
```diff
@@ -1,5 +1,6 @@
-#! /usr/bin/perl -w
+#!/usr/local/env perl
```
Comment (nsoranzo) [inline tools/emboss_5/emboss_infoseq_wrapper.pl]:
```suggestion
#!/usr/bin/env perl
```

File: tools/emboss_5/emboss_multiple_outputfile_wrapper.pl [hunk 3]
```diff
@@ -1,5 +1,6 @@
-#! /usr/bin/perl -w
+#!/usr/local/bin/env perl
```
Comment (nsoranzo) [inline tools/emboss_5/emboss_multiple_outputfile_wrapper.pl]:
```suggestion
#!/usr/bin/env perl
```

File: tools/emboss_5/emboss_single_outputfile_wrapper.pl [hunk 4]
```diff
@@ -1,5 +1,6 @@
-#! /usr/bin/perl -w
+#!/usr/local/bin/env perl
```
Comment (nsoranzo) [inline tools/emboss_5/emboss_single_outputfile_wrapper.pl]:
```suggestion
#!/usr/bin/env perl
```

File: tools/emboss_5/emboss_infoseq_wrapper.pl [hunk 5]
```diff
@@ -1,5 +1,6 @@
-#! /usr/bin/perl -w
```
Comment (nsoranzo) [inline tools/emboss_5/emboss_infoseq_wrapper.pl]: This file is unused, can just be removed.

File: tools/emboss_5/emboss_multiple_outputfile_wrapper.pl [hunk 6]
```diff
@@ -1,5 +1,6 @@
-#! /usr/bin/perl -w
```
Comment (nsoranzo) [inline tools/emboss_5/emboss_multiple_outputfile_wrapper.pl]: This file is unused, can just be removed.

**Comment by nsoranzo** [comment]
@bernt-matthias I meant that those 2 files are unused and can be removed all together, not removing the shebangs!

**Comment by bernt-matthias** [comment]
True, but the shebang line is also unused since the scipts are called: perl '$__tool_directory__/emboss_cpgplot_wrapper.pl'

File: tools/emboss_5/macros.xml [hunk 7]
```diff
@@ -3,6 +3,7 @@
     <xml name="requirements">
         <requirements>
             <requirement type="package" version="@VERSION@">emboss</requirement>
+            <requirement type="package" version="5.26">perl</requirement>
```
Comment (mvdbeek) [inline tools/emboss_5/macros.xml]: hmmm:
```
[Dec  5 12:17:41] STEP Run image [continuumio/miniconda3:latest] with command [[/bin/sh -c conda install  -c conda-forge -c bioconda  emboss=5.0.0 perl=5.26 -p /usr/local --copy --yes --quiet]]
[Dec  5 12:17:41] DEBU Creating container [step-2c22e698d9]
[Dec  5 12:17:41] DEBU Created container [cdeb4e641bf4 step-2c22e698d9], starting it
[Dec  5 12:17:41] DEBU Container [cdeb4e641bf4 step-2c22e698d9] started, waiting for completion
[Dec  5 12:17:49] SOUT Collecting package metadata (current_repodata.json): ...working... done
[Dec  5 12:17:49] SOUT Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
[Dec  5 12:18:11] SOUT Collecting package metadata (repodata.json): ...working... done
[Dec  5 12:18:11] SOUT Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
[Dec  5 12:18:11] SERR 
[Dec  5 12:18:11] SERR PackagesNotFoundError: The following packages are not available from current channels:
[Dec  5 12:18:11] SERR 
[Dec  5 12:18:11] SERR - emboss=5.0.0
[Dec  5 12:18:11] SERR 
[Dec  5 12:18:11] SERR Current channels:
[Dec  5 12:18:11] SERR 
[Dec  5 12:18:11] SERR - https://conda.anaconda.org/conda-forge/linux-64
[Dec  5 12:18:11] SERR - https://conda.anaconda.org/conda-forge/noarch
[Dec  5 12:18:11] SERR - https://conda.anaconda.org/bioconda/linux-64
[Dec  5 12:18:11] SERR - https://conda.anaconda.org/bioconda/noarch
[Dec  5 12:18:11] SERR - https://repo.anaconda.com/pkgs/main/linux-64
[Dec  5 12:18:11] SERR - https://repo.anaconda.com/pkgs/main/noarch
[Dec  5 12:18:11] SERR - https://repo.anaconda.com/pkgs/r/linux-64
[Dec  5 12:18:11] SERR - https://repo.anaconda.com/pkgs/r/noarch
```

Comment (mvdbeek) [reply tools/emboss_5/macros.xml]: oh, it's in the iuc channel :)

**Pull request merged**